docs/install-components/karmada.md

```diff
@@ -0,0 +1,178 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Install Karmada](#install-karmada)
+  - [Prerequisites](#prerequisites)
+    - [Script](#script)
+    - [KinD](#kind)
+    - [kubeadm](#kubeadm)
+  - [Deploy Karmada](#deploy-karmada)
+  - [Add kubernetes cluster to karmada control plane](#add-kubernetes-cluster-to-karmada-control-plane)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Install Karmada
+
+The documentation uses `Ubuntu 20.04.4 LTS` as an example.
+
+## Prerequisites
+
+Installing Karmada requires a kubernetes cluster. If you do not have a kubernetes cluster, you can choose any of the following methods to deploy the kubernetes cluster.
+
+### Script
+Deploy a kubernetes cluster using kurator's scripts. This script will create three clusters for you, one is used to host Karmada control plane and the other two will be joined as member clusters.
+```bash
+git clone https://github.com/kurator-dev/kurator.git
+cd kurator
+hack/local-dev-setup.sh
+```
+
+### KinD
+
+install docker
```

Comment:
I would prefer to link to an official doc if exists.

Reply:
done